graphettes/Graph.cpp

```diff
@@ -288,7 +288,7 @@ void Graph::printStats(int numConnectedComponentsToPrint, ostream& stream) const
 }
 
 void Graph::getDistanceMatrix(vector<vector<short> >& dist) const {
-    string distMatrixFile = "networks/"+name+"/autogenerated/"+name+"_distMatrix.txt";
+    string distMatrixFile = "autogenerated/serialized/"+name+"_distMatrix.txt";
     if (fileExists(distMatrixFile)) {
```

Comment:
File existing isn't enough. Need instead to write new function that also checks if the original network file is more recent than the distance matrix file, and if so, it needs to be recomputed.

(Actually I'm not sure what the files in the graphettes directory does, but the above is generally true; if the original source for some data has changed, any binary or cached one needs to be recomputed.)